docs/releases/1.10.txt

```diff
@@ -505,6 +509,10 @@ Miscellaneous
   argument of the ``render_options()`` method is also removed, making
   ``selected_choices`` the first argument.
 
+* On Oracle/GIS, the :class:`django.contrib.gis.db.models.functions.Area`
+  aggregate function used to return a ``decimal.Decimal`` value (of square
```

Comment:
Thanks. Changed to:
now returns float instead of ``decimal.Decimal`` (it is still wrapped in a measure of square meters).